### ISSUE TITLE
Identify Healthcare investments

### ISSUE THREAD

**Issue by banjtheman** [banjtheman/defundthepolice#5]
# What is the Task
Define a metric that can be used for Healthcare

## Why do we want to do this
To have a social card for Healthcare investments

## How can I get started?
1. Would google how healthcare investments empower communities and how that translates to dollar amounts.
2.   Create a new case in make_investment_image for Healthcare

https://github.com/banjtheman/defundthepolice/blob/main/main_st.py#L64

Here is some pseudo-code
```
    if investment == "Healthcare":
        METRIC_COST= XXX
        WHAT_WE_GAIN= int(math.ceil( reinvest_money / METRIC_COST))

        WHAT_WE_GAIN_STRING= str(f'{WHAT_WE_GAIN:,}')
        text = "That translates to "+ WHAT_WE_GAIN_STRING+ " X for our community" 
        wrapped_string = textwrap.wrap(text, width=30)
        image = draw_image(wrapped_string,bg_color,text_color,font)
        st.image(image, use_column_width=True)
 
```

## Definition of Done
Can generate a social card for Healthcare

**Comment by rohan2734** [comment]
hi i would like to work on this issue

**Comment by banjtheman** [comment]
> 
> 
> hi i would like to work on this issue

Awesome, leave any questions you have in the comments

**Comment by rohan2734** [comment]
@banjtheman  sorry, idk python. i though it is js.

**Comment by Alig1493** [comment]
@banjtheman I found this info:
```
The average national monthly non-subsidized health insurance premium for one person on a benchmark plan (i.e., “Silver” plan) is $462 per month, or $199 with a subsidy. Monthly premiums for ACA Marketplace plans vary by state and can be reduced by subsidies. Actual cost varies based on your age, location, and health plan selection, 
```
From this [article](https://www.ehealthinsurance.com/resources/affordable-care-act/much-health-insurance-cost-without-subsidy)

Can we implement subsidised/non-subsidised loigcs for this issue?

**Comment by banjtheman** [comment]
Using this metric could set the metric cost to 462 and what we measure as people covered by health insurance 

example code
```
    if investment == "Healthcare":
        METRIC_COST= 462
        WHAT_WE_GAIN= int(math.ceil( reinvest_money / METRIC_COST))

        WHAT_WE_GAIN_STRING= str(f'{WHAT_WE_GAIN:,} people covered by health insurance ')
        text = "That translates to "+ WHAT_WE_GAIN_STRING+ " X for our community" 
        wrapped_string = textwrap.wrap(text, width=30)
        image = draw_image(wrapped_string,bg_color,text_color,font)
        st.image(image, use_column_width=True
```

**Comment by Alig1493** [comment]
fixed by #37
